src/subcommands/publishers.rs

```diff
@@ -48,6 +63,12 @@ pub fn publishers(args: Vec<String>, max_age: std::time::Duration) -> Result<(),
             }
         }
         println!("\nGithub teams are black boxes. It's impossible to get the member list without explicit permission.");
```

Comment:
Similar for this, should it be an `eprintln!`? Theoretically we could store permissions somewhere for a number of teams that have been explicitly granted.